tesseract_motion_planners/ompl/include/tesseract_motion_planners/ompl/profile/ompl_profile.h

```diff
@@ -59,6 +61,7 @@ class OMPLMoveProfile : public tesseract_common::Profile
   using ConstPtr = std::shared_ptr<const OMPLMoveProfile>;
 
   OMPLMoveProfile();
+  OMPLMoveProfile(std::string name, const YAML::Node& config, const tesseract_common::ProfilePluginFactory& /*plugin_factory*/);
```

Comment:
I would also un-comment the plugin_factory in all of the headers but leave in the cpp where necessary.